config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -0,0 +1,45 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 6.0 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
+#
+# Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+# Don't force requests from old versions of IE to be UTF-8 encoded.
+Rails.application.config.action_view.default_enforce_utf8 = false
+
+# Embed purpose and expiry metadata inside signed and encrypted
+# cookies for increased security.
+#
+# This option is not backwards compatible with earlier Rails versions.
+# It's best enabled when your entire app is migrated and stable on 6.0.
+Rails.application.config.action_dispatch.use_cookies_with_metadata = true
```

Comment:
Given this says it's not backwards compatible, do we want it on for the initial deployment of Signon with Rails 6?

Reply:
@cbaines This warning looks scary, but it just means that the app is locked into Rails 6.x once this flag is enabled. Downgrading to Rails 5.x won’t be possible because it won’t understand the new "purpose" field in cookies.

Useful/interesting resource about the Rails 6 new defaults: https://medium.com/@dylansreile/rails-6-0-new-framework-defaults-what-they-do-and-how-to-safely-uncomment-them-586146f371e8.

Reply:
Ok, cool :+1: